src/main/resources/assets/js/application/interviewer/controllers/tdprInterviewerHomeController.js

```diff
@@ -185,22 +189,23 @@ define(['angular', 'application/interviewer/tdprInterviewerModule', 'application
         }
 
         function sendNote(note) {
-            tdprPersonService.updateNote(note).then(function(response) {
+            tdprPersonService.updateNote(note).then(function (response) {
                 $scope.temporaryContent = response.data.description;
                 $scope.noteContent = response.data;
-            }, function(failure) {
-                if(failure.status === 406) {
+            }, function (failure) {
+                if (failure.status === 406) {
                     $scope.temporaryContent = $scope.noteContent.description;
                 }
                 Notification.warning({
                     message: 'Something went wrong with sending your note.',
-                    delay: 2000});
+                    delay: 2000
+                });
             });
         }
 
-        $scope.markSlots = function(slot) {
-            if(!$scope.isRecruiter) {
-                if(isSlotTypeFullOrInit(slot) || $scope.relativeDayNumber < 0) {
+        $scope.markSlots = function (slot) {
+            if (!$scope.isRecruiter) {
+                if (isSlotTypeFullOrInit(slot) || $scope.relativeDayNumber < 0) {
```

Comment:
The `$scope.relativeDayNumber < 0` check is also duplicated across the code - it would be good to extract it into a function with a meaningful name.